charts/stable/tt-rss/values.yaml

```diff
@@ -6,8 +6,11 @@
 #
 
 image:
+  # -- image repository
   repository: ghcr.io/k8s-at-home/tt-rss
+  # -- image pull policy
   pullPolicy: IfNotPresent
+  # -- image tag
   tag: v1.8723.0
 
 strategy:
```

Comment:
The `strategy` key can now be removed since it a default is properly determined in common v3.

Also can you add a `# @default -- See below.` to the `env` key helmdoc?

Reply:
removed strategy and added default.